Airports/EGJB/Stars.txt

```diff
@@ -6,13 +6,13 @@ STAR:EGJB:09:GUR1E:ALD GUR
 STAR:EGJB:27:GUR1F:SKERY GUR
 STAR:EGJB:09:GUR1F:SKERY GUR
 STAR:EGJB:09:GUR1G:ORTAC EVKEX GUR
+STAR:EGJB:27:GUR1X:BEVAV OYSTA GUR
+STAR:EGJB:09:GUR1X:BEVAV OYSTA GUR
 STAR:EGJB:27:GUR1Y:LELNA BRILL GUR
 STAR:EGJB:09:GUR1Z:LELNA EVKEX GUR
 STAR:EGJB:27:GUR3C:CEN SENLO BEVAV OYSTA GUR
 STAR:EGJB:09:GUR3C:CEN SENLO BEVAV OYSTA GUR
 STAR:EGJB:27:GUR2H:BIGNO GUR
 STAR:EGJB:09:GUR2H:BIGNO GUR
-STAR:EGJB:27:GUR2J:KOKOS PERCH OYSTA GUR
```

Comment:
Might be worth archiving these STARs using the # nomenclature, as aircraft may continue to terminate their routes at KOKOS for a short while.